daos-cortx/docs/mpifileutils_setup.md

```diff
@@ -0,0 +1,430 @@
+
+
+# Mount cotainer using dfuse
+
+* Create new directories to mount containers
+
+    ```mkdir /mnt/rajkumar/cont1```
+    
+    ```mkdir /mnt/rajkumar/cont2```
+
+* Mount containers
+
+    ```dfuse -m /mnt/rajkumar/cont1 --pool $pool --cont $cont1```
+
+        [root@ssc-vm-2051 daos]# dfuse -m /mnt/rajkumar/cont1 --pool $pool --cont $cont1
+        fi   WARN src/gurt/fault_inject.c:687 d_fault_inject_init() Fault Injection not initialized feature not included in build
+        fi   WARN src/gurt/fault_inject.c:724 d_fault_attr_set() Fault Injection attr not set feature not included in build
+        daos INFO src/client/api/job.c:89 dc_job_init() Using JOBID ENV: DAOS_JOBID
+        daos INFO src/client/api/job.c:90 dc_job_init() Using JOBID ssc-vm-2051.colo.seagate.com-7259
+        mgmt INFO src/mgmt/cli_mgmt.c:360 dc_mgmt_net_cfg() Using client provided OFI_INTERFACE: lo
+        crt  INFO src/cart/crt_init.c:311 crt_init_opt() libcart version 4.9.0 initializing
+        fi   WARN src/gurt/fault_inject.c:687 d_fault_inject_init() Fault Injection not initialized feature not included in build
+        crt  WARN src/cart/crt_init.c:162 data_init() FI_UNIVERSE_SIZE was not set; setting to 2048
+        duns INFO src/client/dfs/duns.c:393 duns_resolve_path() Path does not represent a DAOS link
+        dfuse INFO src/client/dfuse/dfuse_main.c:457 main(0x824000) duns_resolve_path() returned 61 No data available
+        dfuse INFO src/client/dfuse/dfuse_main.c:60 dfuse_send_to_fg() Sending 0 to fg
+
+    ```dfuse -m /mnt/rajkumar/cont2 --pool $pool --cont $cont2```
+
+        [root@ssc-vm-2051 daos]# dfuse -m /mnt/rajkumar/cont2 --pool $pool --cont $cont2
+        fi   WARN src/gurt/fault_inject.c:687 d_fault_inject_init() Fault Injection not initialized feature not included in build
+        fi   WARN src/gurt/fault_inject.c:724 d_fault_attr_set() Fault Injection attr not set feature not included in build
+        daos INFO src/client/api/job.c:89 dc_job_init() Using JOBID ENV: DAOS_JOBID
+        daos INFO src/client/api/job.c:90 dc_job_init() Using JOBID ssc-vm-2051.colo.seagate.com-7284
+        mgmt INFO src/mgmt/cli_mgmt.c:360 dc_mgmt_net_cfg() Using client provided OFI_INTERFACE: lo
+        crt  INFO src/cart/crt_init.c:311 crt_init_opt() libcart version 4.9.0 initializing
+        fi   WARN src/gurt/fault_inject.c:687 d_fault_inject_init() Fault Injection not initialized feature not included in build
+        crt  WARN src/cart/crt_init.c:162 data_init() FI_UNIVERSE_SIZE was not set; setting to 2048
+        duns INFO src/client/dfs/duns.c:393 duns_resolve_path() Path does not represent a DAOS link
+        dfuse INFO src/client/dfuse/dfuse_main.c:457 main(0xa98000) duns_resolve_path() returned 61 No data available
+        dfuse INFO src/client/dfuse/dfuse_main.c:60 dfuse_send_to_fg() Sending 0 to fg
+
+* craete objects (files) inside container
+
+
+    ```cd /mnt/rajkumar/cont1```
```

Comment:
name src_container or source_container instead of cont1

Reply:
done